api.go

```diff
@@ -8,17 +8,20 @@ import (
 	"io/ioutil"
 	"mime/multipart"
 	"net/http"
-	"net/url"
 	"os"
 	"path/filepath"
 	"strconv"
 )
 
-func sendCommand(method, token string, params url.Values) ([]byte, error) {
-	url := fmt.Sprintf("https://api.telegram.org/bot%s/%s?%s",
-		token, method, params.Encode())
+func sendCommand(method, token string, payload interface{}) ([]byte, error) {
+	url := fmt.Sprintf("https://api.telegram.org/bot%s/%s", token, method)
+
+	var b bytes.Buffer
+	if err := json.NewEncoder(&b).Encode(payload); err != nil {
+		return []byte{}, err
```

Comment:
That's a good point, however `return []byte{}, err` was already used in the original code so I kept it consistent with that behavior.